src/patterns/OcFileActions.vue

```diff
@@ -67,18 +78,17 @@ export default {
   <template>
   <div>
     <oc-button uk-toggle="target: #file-actions" text="click to open actions"/>
-    <oc-file-actions id="file-actions"  filename="My file.txt">
-      <oc-file-action-item label="Item 1" @click="onClick" />
-      <oc-file-action-item label="Item 2"/>
-      <oc-file-action-item disabled label="Disabled"/>
-    </oc-file-actions>
+    <oc-file-actions id="file-actions" :items="[{label:'PDF Viewer', icon:'application/pdf', onClick:onPDF},{label:'Text editor', icon:'text/plain', onClick:onTXT}]" filename="My file.txt" />
```

Comment:
for easier reading you could use a data field for the items - just saying

Reply:
@DeepDiver1975 can you point me to an example? I see `model` and `data` being used in the elements but am not sure if thosa are what you mean. I don't see a readable doc for those elements ...

Reply:
https://github.com/owncloud/owncloud-design-system/blob/c856793dd6888602db8cf9477187ab04958b1760/src/patterns/OcAutocomplete.vue#L210